scripts/aspnet.benchmarks.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=ASP.NET Benchmarks Server
+
+# Start after network has at least a bound IP address; stop before network is stopped
+After=network.target
+After=network-online.target
```

Comment:
I think the point is, why does a server need to wait for `network-online.target`?  What's the harm in starting a server before the network is up?